tests/metrics/network/network-metrics-memory-1g.sh.in

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Measures Proportional Set Size memory while running an
+#  inter (docker<->docker) 1Gbps network bandwidth using nuttcp
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Currently default nuttcp has a bug
+# see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=745051
+# Image name
+image=gabyct/nuttcp
```

Comment:
We still don't have the `Dockerfile` for this image in the repository. I've raised #892 to track this as the image is already being used by another test.

This use of custom image is a concern because we cannot recreate it until we have the `Dockerfile`.

In my opinion, we should use standard images where available, but if not, we **must** submit the `Dockerfile` on the same PR as any new tests that use it.

Please can you ensure #892 and the very similar #848 are resolved (it should be just a matter of raising a PR with a single `Dockerfile` for each issue I think?)